CODE_OF_CONDUCT.md

```diff
@@ -0,0 +1,81 @@
+# Code of Conduct
+
+In order to deliver great features and support to the Renovate Open Source project, we ask for high politeness and careful attention from those who want to participate.
```

Comment:
```suggestion
To help us deliver great features and support the Renovate Open Source project we ask that you:

- are polite
- pay attention to details
- keep in mind that most maintainers are volunteers
- are respectful of the time and effort of the maintainers
```

Most people won't read the whole thing. So we should try to put the main points up front. I think this covers most of the big ticket items.